debugoverlay-ext-netstats/src/main/java/com/ms_square/debugoverlay_ext_netstats/NetStatsModule.java

```diff
@@ -0,0 +1,16 @@
+package com.ms_square.debugoverlay_ext_netstats;
+
+import com.ms_square.debugoverlay.OverlayModule;
+import com.ms_square.debugoverlay.modules.SimpleViewModule;
+
+public class NetStatsModule extends OverlayModule<String> {
+
+    private static final int DEFAULT_INTERVAL = 1000; // ms
+
+    public NetStatsModule() {this(DEFAULT_INTERVAL);
```

Comment:
could you move "this(DEFAULT_INTERVAL)" to the next line? It's for better code formatting.